features/pause-audio.js

```diff
@@ -15,7 +15,7 @@ if (window.location.href.startsWith("https://scratch.mit.edu/projects/")) {
           btn2.style.backgroundColor = "#ff9f00";
           btn2.style.marginLeft = ".5rem";
           var img = document.createElement("img");
-          img.src = "https://scratchtools.app/pause.svg";
+          img.src = "https://raw.githubusercontent.com/STForScratch/data/main/pause.svg";
```

Comment:
possibly bundle the svg with the extension? this would result in no fetching of the svg from anywhere

Reply:
Yeah, we plan on doing that soon.